tox.ini

```diff
@@ -0,0 +1,49 @@
+[tox]
+envlist =
+    lint,
+    type
+
+[testenv:lint]
+skipsdist = true
+skip_install = true
+deps =
+    flake8<6
+    flake8-black
+    flake8-bugbear
+    flake8-docstrings
+    flake8-isort
+    flake8-quotes
+commands =
+    flake8 {posargs:.}
+
+[testenv:format]
+skipsdist = true
+skip_install = true
+deps =
+    black
+    isort
+commands =
+    isort {posargs:.}
+    black {posargs:.}
+
+[testenv:type]
+skipsdist = true
+skip_install = true
```

Comment:
We had a chat about this IRL. Removing these lines doesn't work for us right now because we don't have a package to install. If `tifftools` had type annotations we could add it under `deps`, but for now we can leave this as is.